src/App.js

```diff
@@ -1,9 +1,34 @@
+import React from 'react';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import Header from './components/Header/Header';
+import Splash from './components/Splash/Splash';
+import Rooms from './components/Rooms/Rooms';
+import AddRoom from './components/AddRoom/AddRoom';
+import DeleteRoom from './components/DeleteRoom/DeleteRoom';
+import Reservations from './components/Reservations/Reservations';
+import Reserve from './components/Reserve/Reserve';
+import Detail from './components/Detail/Detail';
+import SignUp from './components/SignUp/SignUp';
+import Login from './components/Login/Login';
```

Comment:
Awesome tip @anagudelogu, But my worry when it comes to changing the baseURL is that it might conflict when we need to import resources outside the component. Which we will do when it comes to Redux.